k8s/Makefile

```diff
@@ -59,6 +59,15 @@ export NEW_RELIC_LABELS ?= Project:developer-portal;Namespace:${K8S_NAMESPACE};E
 
 export GOOGLE_ANALYTICS ?= 0
 
+# Backup config
+export RDS_BACKUP_DBTYPE ?= 'PGSQL'
+export RDS_BACKUP_SCHEDULE ?= '@daily'
+export RDS_BACKUP_IMAGE ?= mdnwebdocs/mdn-rds-backup
```

Comment:
that sounds very MDN-specific - is it definitely right?

Reply:
Yeah the image is a generic rds backup script 

See: https://github.com/mdn/infra/tree/master/apps/mdn/utils/rds_backup/image